ngrok/src/config/http.rs

```diff
@@ -49,6 +58,18 @@ pub enum Scheme {
     HTTPS,
 }
 
+impl FromStr for Scheme {
+    type Err = InvalidSchemeString;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        use Scheme::*;
+        Ok(match s {
+            "HTTP" => HTTP,
+            "HTTPS" => HTTPS,
+            _ => return Err(InvalidSchemeString(s.into())),
+        })
```

Comment:
Should we `.to_lowercase()` the input string and match against that? I think `hTTp` is *technically* valid, even if it's unlikely to be used in practice.

Reply:
Pushed case-insensitive change, added to test.